src/mango/src/mango_native_proc.erl

```diff
@@ -47,7 +47,8 @@
 
 
 start_link() ->
-    gen_server:start_link(?MODULE, [], []).
+    throw({error, mango_native_proc_is_no_longer_needed}).
+%%    gen_server:start_link(?MODULE, [], []).
```

Comment:
Don't include commented out code in commits. Also this entire module should just be deleted.